easybuild/easyconfigs/c/CVS/CVS-1.11.23-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,43 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'CVS'
+version = '1.11.23'
+
+homepage = 'https://savannah.nongnu.org/projects/cvs'
+description = """CVS is a version control system, an important component of
+Source Configuration Management (SCM).
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+source_urls = [' http://ftp.gnu.org/non-gnu/cvs/source/stable/%(version)s/']
```

Comment:
@SebastianAchilles is there an `https` version of this?

Reply:
Yes, `https` works! Changed in https://github.com/easybuilders/easybuild-easyconfigs/pull/14283/commits/134b3a7fc21e3592177ed79154a15d030e4c3e62 Thanks for spotting this 👍